card/src/main/java/com/adyen/checkout/card/CardValidationUtils.kt

```diff
@@ -38,18 +39,26 @@ object CardValidationUtils {
     private const val MAXIMUM_YEARS_IN_FUTURE = 30
     private const val MAXIMUM_EXPIRED_MONTHS = 3
 
+    private const val CPF_DIGIT_LIMIT = 11
```

Comment:
Since Marc said this will also be added to Pix in the future, maybe it makes sense to separate the SSN validation outside of the Card to it's own utility class?